menas/src/main/scala/za/co/absa/enceladus/menas/models/SchemaApiAvailability.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.models
+
+case class SchemaApiAvailability(registry: Boolean)
```

Comment:
As you've mentioned this could be generalized for getting availability of potentially many other API features.
Something like `case class RestApiFeatures(isSchemaRegistryAvailable: Boolean)`.

It is a not strong suggestion. Please, feel free to leave it as is if you prefer.

Reply:
While I agree with the `-Features` part, for now, the API is specific for the Schema manipulation, I fear that `RestApi-` is too broad for the usage we have now. We can definitely rename it when it grows and gets used elsewhere.

So, I went with a middle option: `SchemaApiFeature`.